compiler/tests/success/x86-64/opsizes.jazz

```diff
@@ -6,9 +6,9 @@ u = v;
 x = x + s;
 x = x + 64;
 x = x + 64s s;
-x = x >> u;
-x = x >>s 64;
-x = x >>64s u;
+x >>= u & 63;
+x = x >>s 24;
```

Comment:
I think it is good to add this line.
I also think we should add a line close to the previous  one, i.e : x = x >>s 64 & 63;

Reply:
Are you sure?

Reply:
I think constant prop should remove the &63. I is good to check that nothing bad occurs in that case.

Reply:
I agree that it is really an artificial program, but we can imagine something like:
   inline int i;
   for i = 0 to 128 {
       t[i] = t[i] >>s i & 63;
   }

Reply:
Cool, thank